client/src/pages/Home.js

```diff
@@ -58,9 +62,29 @@ function Home({ onPaste }) {
         value={expireTime}
         onChange={event => setExpireTime(parseInt(event.target.value))}
       ></SelectTime>
+
+      <EncryptCheckbox
+        value={checkbox}
+        label={'Encrypt my Pasta'}
```

Comment:
The other components on this page also have their label within the functional component file so I think this label can also be moved to the `EncryptCheckbox.js` file.

Reply:
you're right, but in this case we have a practical component that can be used with others labels repeatedly...